src/middlewared/middlewared/pytest/unit/utils/test_statx.py

```diff
@@ -96,7 +96,7 @@ def test__check_statx_empty_path(tmpdir):
     stx1 = sx.statx(testfile)
     try:
         fd = os.open(testfile, os.O_PATH)
-        stx2 = sx.statx('', {'dir_fd': fd, 'flags': sx.ATFlags.EMPTY_PATH})
+        stx2 = sx.statx('', dir_fd=fd, flags=sx.ATFlags.EMPTY_PATH)
```

Comment:
Should this be `sx.ATFlags.EMPTY_PATH.value`

Reply:
Either works since it's IntFlag enum and not performance critical.